### ISSUE TITLE
Disable autoplay of videos using an add-on

### ISSUE THREAD

**Issue by devcer** [firefox-addons/ideas#28]
Not sure if we have this. If didn't found similar addon then i will start working on this.

**Comment by Mte90** [comment]
I think that actually Firefox do it automatically

**Comment by tsl143** [comment]
@viswanathamsantosh, Does this helps https://addons.mozilla.org/en-US/firefox/addon/simple-youtube-controller/

**Comment by devcer** [comment]
Yes. but may be we can extend it to all HTML5 video players? Let me have a look at this.
